.github/workflows/build.yml

```diff
@@ -28,5 +28,8 @@ jobs:
       - name: Build
         run: |
           go install github.com/u-root/mkuimage/cmd/mkuimage@latest
+          go install github.com/u-root/mkuimage/cmd/mkuimage@latest
```

Comment:
now this line is duplicated... sorry 🙃

Reply:
wth. That's not what is on my local malchine.